main.js

```diff
@@ -1,118 +1,370 @@
-
-//* Task #3
-const arenas = document.querySelector(".arenas");
+/**
+ * Функция создает элемент HTML
+ * @param {string} tag
+ * @param {string} className
+ * @returns {HTMLElement}
+ */
+function createElement(tag, className) {
+	const $tag = document.createElement(tag);
+
+	if (className) {
+		$tag.classList.add(className);
+	}
 
+	return $tag;
+}
+/**
+ * Функция создает игрока
+ * @param {object} player объект игрока
+ * @param {string} player.classTitle класс игрока
+ * @param {string} player.name имя игрока
+ * @param {number} player.hp жизни игрока
+ * @param {string} player.img аватар игрока
+ * @returns {HTMLElement}
+ */
 function createPlayer(player) {
-	let { title, name: playerName, hp, img } = player;
-	const $player = document.createElement("div");
-	$player.classList.add(title);
+	const playerEl = createElement('div', player.classTitle);
+	const progressbar = createElement('div', 'progressbar');
+	const life = createElement('div', 'life');
+	const name = createElement('div', 'name');
+	const character = createElement('div', 'character');
+	const image = createElement('img');
+	const hpPlayer = createElement('div', 'hpPlayer');
+	name.innerText = player.name;
+	life.style.width = `${player.hp}%`;
+	hpPlayer.innerText = `${player.hp}%`;
+	image.src = player.img;
+	character.append(image);
+	progressbar.append(hpPlayer, name, life);
+	playerEl.append(progressbar, character);
+	return playerEl;
+}
+/**
+ * Функция выводит результат игры, победа или ничья
+ * @param {string} name
+ * @returns {HTMLElement}
+ */
+function getPlayerWins(name) {
+	const gameResult = createElement('div', 'gameResult');
+	if (name) {
+		gameResult.innerText = name + ' wins';
+	} else {
+		gameResult.innerText = 'Draw';
+	}
 
-	const progressbar = document.createElement("div");
-	progressbar.classList.add("progressbar");
+	return gameResult;
+}
+/**
+ * Функция генерации случайных чисел в диапазоне min / max
+ * @param {number} min
+ * @param {number} max
+ * @returns {number}
+ */
+function getRandom(min, max) {
+	let rand = min + Math.random() * (max + 1 - min);
+	return Math.floor(rand);
+}
 
-	const life = document.createElement("div");
-	life.style.width = `${hp}%`;
-	const name = document.createElement("div");
-	life.classList.add("life");
-	name.classList.add("name");
-	name.innerText = playerName;
+/**
+ * Функция уменьшает HP у игрока на величину damage
+ * @param {number} damage
+ * @returns {number}
+ */
+function changeHP(damage) {
+	this.hp -= damage;
+}
+/**
+ * Функция возвращает элемент здоровья персонажа
+ * @param {string} classElem обращаемся к элементу в котором происходит рендеринг hp
+ * @returns {HTMLElement}
+ */
+function elHP(classElem = 'life') {
+	return document.querySelector(`.${this.classTitle} .${classElem}`);
+}
 
-	const character = document.createElement("div");
-	character.classList.add("character");
+/**
+ * Функция отрисовывает кол-во жизней в %
+ * @returns {HTMLElement}
```

Comment:
ничего не возвращает
```suggestion
 * @returns {void}
```